### PR TITLE
Add ability to exclude rule from counting

### PR DESCRIPTION
**Pull request by Vincent056**
In rule's e2e.yaml file, you can now set ExcludeFromCount to any value to exclude the rule from counting.

### CODE REVIEW

File: helpers.go [hunk 1]
```diff
@@ -932,6 +938,16 @@ func verifyRuleResult(
 	return nil
 }
 
+// Will exclude the rule from counting if excludedString has values
+func isExcluded(exclude interface{}) bool {
+	if excludedString, ok := exclude.(string); ok {
+		if excludedString != "" {
```
Comment (jhrozek) [inline helpers.go:944]: are we returning true on any value to support true/TRUE/yes/YES/on/ON? Was wondering what would happen if someone says "exclude_from_count: FALSE" in the content..

**Review by jhrozek — COMMENTED** [review]
One question, mostly looks good and I'm glad you found a way around the issue

File: helpers.go (hunk 1 above)
Comment (Vincent056) [reply helpers.go:944]: Let's change it to `Yes`

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**